src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$transition-color: color 300ms ease-in-out;
```

Comment:
You could use a mixin, but not the exact value with exact transition-property value